vectorizers/_vectorizers.py

```diff
@@ -258,12 +258,21 @@ def preprocess_token_sequences(
         index: token for token, index in token_dictionary.items()
     }
 
-    result_sequences = [
+    """
+     result_sequences = [
         np.array(
             [token_dictionary[token] for token in sequence if token in token_dictionary]
         )
         for sequence in token_sequences
     ]
+    """
+    result_sequences = []
+    for sequence in token_sequences:
+        result_sequence = np.array(
+            [token_dictionary[token] for token in sequence if token in token_dictionary]
+        )
```

Comment:
```suggestion
        result_sequence = np.array(
            [token_dictionary[token] for token in sequence if token in token_dictionary],
            dtype=np.int64,
        )
```
I think you want the above, as it will ensure homogeneous types. If you drop sequences entirely won't that mean that the resulting array of document vectors, for example, may no no match with the inputs and your indexing will be all screwy?

Reply:
Good call.  I changed it.